karma.config.js

```diff
@@ -5,7 +5,7 @@ const seenBrowsers = {};
 const isTravis = process.env.TRAVIS;
 // Browser detection does not work properly on Travis.
 const installedBrowsers = isTravis ? ['Firefox'] : detectBrowsers.getInstalledBrowsers()
-  .map(function(browser) { return browser.name; })
+  .map(function(browser) { return browser.name.replace(/ /, ''); })
```

Comment:
Can you add a comment explaining the bug above the fix and then squash the commits?

e.g.

```javascript
// XXX: Browsers with spaces in their name (e.g. Chrome Canary) need to have the space removed
```